assets/mobile-nav.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
This file, and the attach-event-to-dash are the ones which could/should be moved to the React library at some point.